src/components/ContinuationIn/ContinuationAuthorization.vue

```diff
@@ -403,7 +406,7 @@ textarea {
   // ref: https://developer.mozilla.org/en-US/docs/Web/CSS/field-sizing
   // field-sizing: content;
   // unset height as it's sometimes 0 initially
-  height: unset !important;
```

Comment:
`!important` overrode the component's own height specification (set by the v-auto-resize directive). It works perfectly with the new code.